<launch>

<!-- Launch HSR navigation node --> 
<node pkg="hsr_navigation" type="planner" respawn="false" name="hsr_navigation" output="screen">
    <rosparam file="$(find hsr_navigation)/params/costmap_common_params.yaml" command="load" ns="global_costmap" /> 
    <rosparam file="$(find hsr_navigation)/params/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find hsr_navigation)/params/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find hsr_navigation)/params/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find hsr_navigation)/params/hsr_dwa_planner.yaml" command="load" />
</node>

</launch>

